src/Microsoft.Sbom.Extensions.DependencyInjection/ServiceCollectionExtensions.cs

```diff
@@ -46,29 +45,11 @@ namespace Microsoft.Sbom.Extensions.DependencyInjection;
 
 public static class ServiceCollectionExtensions
 {
-    public static IServiceCollection AddSbomConfiguration(this IServiceCollection services, InputConfiguration inputConfiguration, LogEventLevel logLevel = LogEventLevel.Information)
-    {
-        ArgumentNullException.ThrowIfNull(inputConfiguration);
-        services
-            .AddSingleton(_ =>
-            {
-                inputConfiguration.ToConfiguration();
-                return inputConfiguration;
-            })
-            .AddSbomTool(logLevel);
-        return services;
-    }
-
-    public static IServiceCollection AddSbomTool(this IServiceCollection services, LogEventLevel logLevel = LogEventLevel.Information)
+    public static IServiceCollection AddSbomTool(this IServiceCollection services)
     {
         services
             .AddSingleton<IConfiguration, Configuration>()
-            .AddTransient(_ => FileSystemUtilsProvider.CreateInstance(CreateLogger(logLevel)))
-            .AddTransient(x =>
```

Comment:
My understanding is that this will create a breaking change for API users, as they will now have to inject a logger similarly to how we are now doing it in Program.cs, is that right? This is okay with me as our next release will be a breaking change anyways, but tagging @DaveTryon for visibility

Reply:
Oof, good call - I didn't realize that the dependency injection project was public API surface area. Let me noodle on how that could be fixed.

Reply:
I've pushed a commit that minimizes the API surface area changes, and adds a ton of xmldoc that instructs users on the new requirement of registering their own `ILogger`. That won't stop someone that just updates and then runs tests from failing, but maybe that will help mitigate somewhat.

Reply:
@Forgind is there a way we can avoid making a breaking change here?

Reply:
baronfel didn't come up with a way, and he knows this part much better than I do. I can try to look around, but I have little hope of success.